src/main/java/controllers/UserProfileController.java

```diff
@@ -65,8 +62,9 @@ public void runBrowseProfile() {
         // display all the usernames in the database
         this.runDisplayAllUsers();
         String userToBrowse = this.inOut.getInput("Enter the username of the person you'd like to view: ");
-        if (this.userManager.checkUserToBrowse(userToBrowse) &&
-                !(userToBrowse.equals(this.loginManager.getCurrUser().getUsername()))) {
+        // if ((this.userManager.checkUserToBrowse(userToBrowse) &&
```

Comment:
If we aren't using this code we should delete it, not comment it out.

Reply:
sounds good